File: fetch.js
#!env node
const fs = require("fs");

const query = fs.readFileSync(__dirname + "/query.sparql").toString();

const url = `https://query.wikidata.org/sparql?query=${encodeURIComponent(
  query
)}`;

fetch(url, {
  headers: { Accept: "application/sparql-results+json" },
})
  .then((r) => r.text())
  .then(console.log);

File: wikidata/fetch.py
#!env python3
import requests

with open("query.sparql", "r") as file:
    query = file.read()

res = requests.get(
    "https://query.wikidata.org/sparql",
    params={
        "query": query,
    },
    headers={"Accept": "application/sparql-results+json"},
)

print(res.text)
